mailbox/api/src/main/java/org/apache/james/mailbox/model/Attachment.java

```diff
@@ -107,6 +107,10 @@ public InputStream getStream() throws IOException {
         return new ByteArrayInputStream(bytes);
     }
 
+    public byte[] getBytes() {
+        return Arrays.copyOf(bytes, bytes.length);
```

Comment:
The problem is the data type. We should try to use ByteBuffer : it supports read-only and view creation.